src/prg-4-5.asm

```diff
@@ -1264,10 +1295,15 @@ PlayNote_CheckSquareChorus:
 	CMP #$E0
 	BEQ PlayNote_SetFrequency_Square2Detuned
 
+IFDEF EXPAND_MUSIC
+	LDA MusicSquare1NoteBend, X
+	BNE NoteBendStashFrequency
+ENDIF
+
 PlayNote_SetFrequency:
 	LDA NextFrequencyLo
 	STA SQ1_LO, X
-	STA UNUSED_MusicSquare1Lo, X ; unused?
+	STA MusicSquare1Lo, X ; unused
```

Comment:
idly curious about this change (when `UNINITIALIZED` was left in below)

Reply:
i started to, but i ran into problems with `UNINITIALIZED_MusicSquare2Lo` because there's already `MusicSquare2Lo`. i think one of them might be misnamed, but i didn't spend the time to figure it out yet 😅

Reply:
welp, figures.